src/graphql/queries/arc.graphql

```diff
@@ -564,6 +580,51 @@ query trendingTokens($chainId: Int) {
         }
         updated_at
       }
+      trending {
+        rank
+        trending_since
+        pool_data {
+          currency_used
+          reserve
+          m5_volume
+          h1_volume
+          h6_volume
+          h24_volume
+          m5_price_change
+          h1_price_change
+          h6_price_change
+          h24_price_change
+        }
+        swap_data {
+          currency_used
+          bought_stats {
+            unique_users
+            total_transactions
+            total_volume
+            farcaster_users {
+              fid
```

Comment:
do we need all these?

Reply:
not sure yet but will trim what we don't use in a follow-up